### ISSUE TITLE
Help needed: Unable to propagate click events to components inside the coulmn header

### ISSUE THREAD

**Issue by talamaska** [handsontable/handsontable#6071]
### Description
So, what I'm trying to do is the same what I did for the cell renderer, I created a logic inside the colHeaders Function to instanciate angular components, that have their own click events. For example for each row in the last column I'm rendering a drop down button and that's working fine. But the click events in the components injected in the column header are not triggering. 
Would you like to give me some direction. I saw the examples of vanila js and vanila eventListeners but, this is not helping me any way. Basically no events are not propagating to the component, mouse over, mouse out, click.
What I'm missing, obviously there is something special about the col headers.

### Steps to reproduce
complicated
basically using the componentResolveFactory to create angular component on the fly.
then appending that to the col header. the component is injected succesfully, but no click are happening

### Demo
Sorry my code is properiatery, very difficult to recreate a simplified version

### Your environment
* Handsontable version: 7.0.3
* Browser Name and version: Chrome 69
* Operating System: Windows 10

**Comment by AMBudnik** [comment]
Hey @talamaska 

that is a pretty custom case that I haven't had a chance to debug or try. 
Do all the click events for the headers are lost or you're able to get the `e.target` from the [`beforeOnCellMouseDown`](https://handsontable.com/docs/7.1.0/Hooks.html#event:beforeOnCellMouseDown)?

**Comment by talamaska** [comment]
Even if there is e.target, the click doesn't trigger. Sure I can listen to the click to my component via the hook, but that's not the idea.

**Comment by AMBudnik** [comment]
I don't know how can I help here. I would need to get a demo where the issue can be replicable.

**Comment by talamaska** [comment]
I'll create a stackblitz, just been busy before the release we have. Maybe after July 15, is that ok?

**Comment by AMBudnik** [comment]
Yes sure. I can close the issue now as we don't have anything we can test or debug but will reopen it as soon as we get the demo. Is it OK with you?

**Comment by talamaska** [comment]
ok
